stages/org.osbuild.copy

```diff
@@ -45,9 +45,18 @@ SCHEMA_2 = r"""
         "required": ["from", "to"],
         "properties": {
           "from": {
-            "type": "string",
-            "description": "The source",
-            "pattern": "^input:\/\/[^\/]+\/"
+            "oneOf": [
+              {
+                "type": "string",
+                "description": "The source, if an input",
+                "pattern": "^input:\/\/[^\/]+\/"
+              },
+              {
+                "type": "string",
+                "description": "The source, if the tree",
+                "pattern": "^tree:\/\/\/"
```

Comment:
I think what we really want is:  `"pattern: "^(input|mount):\/\/[^\/]+\/"` So we would copy from the mount point where we have the ostree already mounted

Reply:
in this case I really do want to copy from the tree. but you bring up a great point.. why not allow copying `from:` a `mount:///` too?

Reply:
We can do so, you can either add it to this PR or it can be done separately when it comes up again.